support-frontend/assets/components/button/_sharedButton.jsx

```diff
@@ -14,6 +14,8 @@ import './button.scss';
 export const Appearances = {
   primary: 'primary',
   secondary: 'secondary',
+  tertiary: 'tertiary',
```

Comment:
What are 'appearances'?

Reply:
This is about the type of look the button has ('primary' is yellow, 'secondary' is solid back with white text, etc).